docs/qa/method.md

```diff
@@ -0,0 +1,182 @@
+---
+order: 1
+title: Method
+---
+
+The (first iteration of the) QA process as described [here][releases] is applied to version v0.34.x
+in order to have a set of results acting as benchmarking baseline.
+This baseline is then compared with results obtained in later versions.
+
+Out of the testnet-based test cases described in [releases][releases] we are focusing on two of them:
+_200 Node Test_, and _Rotating Nodes Test_.
+
+[releases]: https://github.com/tendermint/tendermint/blob/v0.37.x/RELEASES.md#large-scale-testnets
+
+# Software Dependencies
+
+## Infrastructure Requirements to Run the Tests
+
+* An account at Digital Ocean (DO), with a high droplet limit (>202)
+* The machine to orchestrate the tests should have the following installed:
+    * A clone of the [testnet repository][testnet-repo]
+        * This repository contains all the scripts mentioned in the reminder of this section
+    * [Digital Ocean CLI][doctl]
+    * [Terraform CLI][Terraform]
+    * [Ansible CLI][Ansible]
+
+[testnet-repo]: https://github.com/interchainio/tendermint-testnet
+[Ansible]: https://docs.ansible.com/ansible/latest/index.html
+[Terraform]: https://www.terraform.io/docs
+[doctl]: https://docs.digitalocean.com/reference/doctl/how-to/install/
```

Comment:
That testnet repo is currently private, I'm fine making it public, do you think that's the right course of action?

Reply:
Let's leave it as it is ATM (i.e., private). This is here for reproducibility purposes, people can always ask access.
When we decide what to do with the repo (make it public?, move it inside Tendermeint repo?, etc) then we can come back here and adapt the link.

... for the time being we can add a comment here like: "private repo, ask Tendermint Core Team for access if you are interested". What do you think?